src/Illuminate/Foundation/Exceptions/Handler.php

```diff
@@ -208,6 +216,20 @@ public function ignore(string $class)
         return $this;
     }
 
+    /**
+     * Set the log level for the given exception type.
+     *
+     * @param  class-string<\Throwable>  $type
+     * @param  string  $level
```

Comment:
`string` could be replaced by `LogLevel::*`

Reply:
#41935